### DISCUSSION TITLE
osmand sync tracks; can't see data copied to tracks folder

### DISCUSSION THREAD

**Discussion by fritzphantom** [osmandapp/OsmAnd#19080] [Q&A]
Hello, not sure if this is the rigth place to ask, but anyway: I would like to sync my tracks in osmand to google drive. For this i need a folder that is accessible by both the sync app and osmand. I choose now manually: /storage/emulated/0/Documents/osmand
which puts all osmand data there, and inside is a folder called tracks.

The issue is that osmand does not see tracks that are copied there manually or by the sync app. Does anybody know why not? It is Android 13.

Thansk!

**Comment by mariush444** [comment]
You have to use .../Android/media/net.osmand.plus/files/

**Comment by fritzphantom** [comment]
Yes that worked, thanks! Is that actually somewhere in the documentation?

**Comment by mariush444** [comment]
I Don't think so. It related to Android (problems.ideas)
